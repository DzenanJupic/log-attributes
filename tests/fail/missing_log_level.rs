#[log_attributes::log("{fn} was executed")]
fn missing_log_level() {}

fn main() {}
