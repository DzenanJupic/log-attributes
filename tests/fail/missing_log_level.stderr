error: Expected log level
       note: The log level should be provided as an identifier without quotes
       example: #[log(info, "{fn} returned {return}")]
 --> tests/fail/missing_log_level.rs:1:23
  |
1 | #[log_attributes::log("{fn} was executed")]
  |                       ^^^^^^^^^^^^^^^^^^^
